### PR TITLE
Created a positive/negative test case for a UDID.

### PR DESCRIPTION
**Pull request by chelseavuata**

### CODE REVIEW

**Review by Lesterconey — DISMISSED** [review]
Looks good.

File: test_team_code_busters.py [hunk 1]
```diff
@@ -6,3 +6,16 @@ class TestTeamCodeBusters(unittest.TestCase):
...
+        '''Testing if UDID is detected'''
+
+        # positive test case
+        results = analyze_text('A7C9B2A4-F6E82B43C580E24F')
+        print(results)
+        self.assertIn('UDID', str(results))
+
+        # negative test case
+        results = analyze_text('K7Y9J2A4-R6T8X1')
+        print(results)
```
Comment (tolanism) [inline test_team_code_busters.py:20]: Looks good to me. Nice!

**Comment by tolanism** [comment]
I like it!

**Review by tolanism — APPROVED** [review]

**Review by claesmk — COMMENTED** [review]
The code could be OK but I can't approve it because it does not match the description in your [issue](https://github.com/NCATComp410/comp410_fall_2023/issues/33)

```
It is a 40-digit sequence of letters and numbers which can be used for detection.
```

Your are detecting a 8+16 = 24 which does not match the 40-digit in your description.  You were asked to provide a sample number in the issue and that was never provided. 

Please update your issue to match your code

**Comment by claesmk** [comment]
@chelseavuata please re-request a code review after you feel your issue is updated and ready for me to review again

**Review by claesmk — APPROVED** [review]
Requirement in the issue now matches the implementation - approved

**Pull request merged**